scipy/signal/lti_conversion.py

```diff
@@ -384,6 +384,36 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
     approximation is based on [2]_ and [3]_, the First-Order Hold (foh) method
     is based on [4]_.
 
+    Examples
+    --------
+    We can transform a continuous state-space system to a discrete one:
+
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy.signal import cont2discrete, lti, dlti, dstep
+
+    Define a continious state-space system.
+
+    >>> A = np.array([[0, 1],[-10, -3]])
+    >>> B = np.array([[0],[10]])
+    >>> C = np.array([[1, 0]])
+    >>> D = np.array([[0]])
+    >>> l_system = lti(A, B, C, D)
+    >>> t, x = l_system.step(T=np.linspace(0,5,100))
+    >>> plt.plot(t, x, label='Continuous')
```

Comment:
Maybe this is not visible enough. Also there is no label for the `hlines`. All in all a bit more context would help I guess.

Reply:
OK. I removed the `hlines` and changed the line size. PTAL.